.travis.yml

```diff
@@ -15,10 +15,9 @@ install:
 
   # install dependencies
   - source $HOME/miniconda/etc/profile.d/conda.sh
-  - conda create -q -n drop -c conda-forge -c bioconda python=$TRAVIS_PYTHON_VERSION r-base=4.0.2
+  - conda create -q -n drop -c conda-forge -c bioconda "python>=3.8" "r-base>=4.0.3" "drop>=1.0.1" "wbuild>=1.8" "bioconductor-fraser>=1.2.0"
```

Comment:
Do we want to force python 3.8 or use the python version specified in the travis variable?

Reply:
@mumichae, I "forced" the python version to speed up the installation. If you do not specify anything, conda uses `3.8`. Hence, I was just following conda. Forcing it to the travis version makes only sense if we test it for multiple python versions.